### ISSUE TITLE
CSS affichage formulaire saisie

### ISSUE THREAD

**Issue by MNourrisson** [PnX-SI/GeoNature-citizen#411]
Bonjour, 

Il y a une différence d'affichage entre la saisie d'une observation et celle d'un site quand le formulaire est en mode popup, en vue ordi.

![obs](https://github.com/PnX-SI/GeoNature-citizen/assets/34370970/c4aac9ba-b195-428f-8224-b313996df87c)
https://github.com/PnX-SI/GeoNature-citizen/blob/68469ff72e48bd0ae47ff668ad701219026efeac/frontend/src/app/programs/observations/form/form.component.html#L139

![site](https://github.com/PnX-SI/GeoNature-citizen/assets/34370970/0a7e5a40-adfe-436e-bf28-8d2ac1c1d6f9)
https://github.com/PnX-SI/GeoNature-citizen/blob/68469ff72e48bd0ae47ff668ad701219026efeac/frontend/src/app/programs/sites/siteform/siteform.component.html#L4
https://github.com/PnX-SI/GeoNature-citizen/blob/68469ff72e48bd0ae47ff668ad701219026efeac/frontend/src/app/programs/sites/siteform/siteform.component.html#L24

Je trouve que l'affichage en deux colonnes n'est pas vraiment pratique pour utiliser la carte.
Serait-il possible d'ajouter un paramètre dans la conf pour choisir entre les deux affichages ?

Version : 1.1.0

Merci

**Comment by xavyeah39** [comment]
Yes carrément ! 
Mais je m'interroge sur la pertinence d'un paramètre pour laisser le choix : c'est carrément plus fonctionnel avec la carte en dessous du form en pleine largeur de la modale ! Sinon on saisie dans un timbre-poste c'est dommage...  
Pour moi il faut juste mettre la carte en dessous pour harmoniser avec la l'agencement de la modale des observations. 

Comme ce que tu as fais @MNourrisson : 
![image](https://github.com/PnX-SI/GeoNature-citizen/assets/16793813/b37fcafa-dbcc-455b-999d-d71782bd5a90)


A moins qu'il y ai un sens particulier a disposer la carte dans une colonne en vis-à-vis des champs ?

**Comment by camillemonchicourt** [comment]
Oui oui il faut faire simple, harmoniser entre programmes espèces et programmes sites (idéalement factoriser le code ?) et ne pas ajouter de paramètre pour cela, selon moi.

**Comment by xavyeah39** [comment]
Oui, effectivement, les composant Map sont actuellement distincts pour sites et observations.

**Comment by lpofredc** [comment]
Fixé par PR #427, cf. version [1.2.0](https://github.com/PnX-SI/GeoNature-citizen/releases/tag/1.2.0)
